docs/getting-started/orchestration_quickstart.md

```diff
@@ -0,0 +1,164 @@
+# Orchestration Quickstart
+
+This guide is desinged to get you to the point of successfully deploying a Prefect flow in as few steps as possible. For a more comprhensive introduction to Prefect's core orchestration concepts, please follow our [tutorial pages](/tutorial/index/).
+
+### Step 1: [Install Prefect](/getting-started/installation/)
+
+We recommend installing Prefect using a Python virtual environment manager such as pipenv, conda, or virtualenv/venv.
+
+```bash 
```

Comment:
Probably want to add <div class=terminal> </div> for bash code snippets for consistency with other docs pages.